dataactbroker/handlers/userHandler.py

```diff
@@ -390,7 +390,7 @@ def getPermssionList(self):
         queryResult = self.session.query(PermissionType).all()
         return queryResult
 
-    def createUserWithPassword(self,email,password,bcrypt,admin=False):
+    def createUserWithPassword(self,email,password,bcrypt,permission=1):
```

Comment:
This can be avoided by calling UserHandler.getPermissionId("agency_user").  The one below would be "website_admin".  If desired, you could refactor this function to take a list of permission strings rather than a single number.  Alternatively, just set the default value here to None, and then in the code:
`if permission is None:
    permission = self.getPermissionId("agency_user")`